tailwind.config.ts

```diff
@@ -7,12 +7,28 @@ const config: Config = {
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
   ],
   theme: {
-    extend: {
-      backgroundImage: {
-        "gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
-        "gradient-conic":
-          "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
-      },
+    extend:{
```

Comment:
There should be one extend object, and colors should also be inside it.

Reply:
The styling codes from line 11 -15  on backgroundImage are not in my project, i dont know how this is showing on my github
I do not have two extend objects 
I have added colors into the extend object, thank you.

Reply:
Please help me review again.